rllib/policy/dynamic_tf_policy_v2.py

```diff
@@ -251,15 +251,12 @@ def compute_gradients_fn(
     @OverrideToImplementCustomLogic
     def apply_gradients_fn(
         self,
-        policy: Policy,
```

Comment:
Why was this here before?

Reply:
Because in the policy builder api `policy` need to be passed in as the first argument, but `self` takes care of it in PolicyV2, and this is probably leftover from copy-paste and was never found since apply_gradients_fn was never used in a PolicyV2 algorithm until now

Reply:
Thanks!